rosdep/python.yaml

```diff
@@ -2824,6 +2824,16 @@ python-multicast:
     pip: [py-multicast]
   ubuntu:
     pip: [py-multicast]
+python-multimethod:
```

Comment:
You probably want to make it available for python3.
It's also a pip package, so the name should be:

```suggestion
python3-multimethod-pip:
```

Reply:
Thank you for pointing out!
I modified in 8c71ccc.